.github/workflows/test_ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 12.x
+          node-version: 14.x
```

Comment:
do we need to even specify here?
i thought we set the `.nvmrc` to be 14.11 already?